licenses.d2iq.yaml

```diff
@@ -22,6 +22,11 @@ resources:
         directory: bitnami/external-dns
         license_path: LICENSE.md
   - container_image: docker.io/bitnami/kubectl:1.24.6
+    sources:
+      - url: https://github.com/bitnami/bitnami-docker-kubectl
```

Comment:
Looks like this repo has been archived and moved to https://github.com/bitnami/containers/tree/main/bitnami/kubectl/1.24